playbooks/vars/onmetal.yml

```diff
@@ -11,7 +11,7 @@ gating_overrides:
   tempest_testr_opts:
     - '--concurrency 3'
   tempest_run_tempest_opts: []
-  tempest_public_subnet_cidr: "172.29.248.0/22"
+  tempest_public_subnet_cidr: "10.0.248.0/22"
```

Comment:
same as above. maybe it would help if we made the base cidrs configurable upstream?